flink-connectors/flink-connector-hive/src/test/resources/query-test/join.q

```diff
@@ -26,7 +26,7 @@ select * from foo left semi join bar on foo.y=bar.i;
 
 select * from (select a.value, a.* from (select * from src) a join (select * from src) b on a.key = b.key) t;
 
-[+I[val1, 1], +I[val2, 2], +I[val3, 3]]
+[+I[val1, 1, val1], +I[val2, 2, val2], +I[val3, 3, val3]]
```

Comment:
Run the sql in Hive CLI, the result is 
`[+I[val1, 1, val1], +I[val2, 2, val2], +I[val3, 3, val3]]`.
So, make the change for the previous is wrong